docs/features/cluster-egress-controls/snat-exclusion.md

```
@@ -0,0 +1,64 @@
+By default, OVN-Kubernetes masquerades (SNATs) pod egress traffic to the node's IP address
+as it leaves the cluster. This is correct for most workloads, but some applications — such as
+[Submariner](https://submariner.io/) — require the original pod source IP to be preserved for
+traffic destined to specific subnets.
+
+OVN-Kubernetes supports a per-node annotation that lets cluster administrators specify subnets
+for which SNAT should be skipped, preserving the pod's original source IP for traffic to those
+destinations.
+
+## Annotation
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)